requirements.txt

```diff
@@ -0,0 +1,71 @@
+agate==1.9.1
```

Comment:
Is this a standard part of dbt projects?

Reply:
removed it entirely